dapr/clients/grpc/client.py

```diff
@@ -315,6 +316,7 @@ def publish_event(
             topic_name (str): the topic name to publish to
             data (bytes or str): bytes or str for data
             metadata (tuple, optional): custom metadata
+            data_content_type: (str, optional): content type of the data payload
```

Comment:
should this made default since it is not set by the SDK?

Reply:
CloudEvent spec treats this as optional:
https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#optional-attributes

As does a comment in our own proto spec.